docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  alice:
+    build: .
+    image: datahighway:latest
+    ports:
+      - 30333:30333
```

Comment:
do we need to expose a range of ports? (i.e. 30333:30344, and 9933:9944)

Reply:
the latest compose file does it but these ports can be changed and then compose file needs updating, there is a note about updating docker-compose.yml and Dockerfiles to suite the need.